components/molecules/BenefitCard.js

```diff
@@ -92,13 +92,13 @@ const BenefitCard = (props) => {
 
   const renderBenefitDetails = () => {
     switch (props.benefit.status.toUpperCase()) {
-      case BenefitStatus.pending.toUpperCase():
-        return (
-          <PendingBenefitDetails
-            benefit={props.benefit}
-            locale={props.locale}
-          />
-        )
+      //   case BenefitStatus.pending.toUpperCase():
```

Comment:
Why have you commented the switch case for Pending card?
There are still difference between details section for Active and Pending cards.